### ISSUE TITLE
Hello there

### ISSUE THREAD

**Issue by thatsparrow** [thatsparrow/blog#1]
Oh, why should people read your blog, you ask? Well, let's see. 

First of all, I'm sure everyone is just dying to hear my thoughts on everything from the latest trends in cat fashion to the philosophical implications of the Oxford comma.

And let's not forget the endless entertainment value I provide. I mean, who doesn't love reading the musings of a complete nobody on the internet? I'm sure everyone is just clamoring to hear about my daily routine and what I ate for breakfast this morning.

And if that's not enough, I'm also pretty sure my blog has the power to cure all ailments and end world hunger. I mean, I haven't actually tested that theory, but I'm pretty confident that my writing is just that good.

So, in conclusion, you should definitely read my blog. I mean, why waste your time on anything else when you could be reading the ramblings of a complete stranger with absolutely no credentials or expertise? It's not like there are better ways to spend your time or anything.
